prod-cockpit-rcar.yaml

```diff
@@ -195,8 +195,12 @@ components:
         - [MACHINE_FEATURES_append, " multimedia"]
 
         # take dedicated cr7 kernel
-        - [RENESAS_BSP_URL_pn-linux-renesas,  "git://git@gitpct.epam.com/rec-inv/linux.git;protocol=ssh"]
-        - [BRANCH_pn-linux-renesas, "v5.10_cr7_aosbox_clusterdrv"]
+        - [RENESAS_BSP_URL_pn-linux-renesas,  "git://git@github.com/xen-troops/linux.git;protocol=ssh"]
+        - [BRANCH_pn-linux-renesas, "cr7-cockpit-demo"]
+        - [SRCREV_pn-linux-renesas, "${AUTOREV}"]
```

Comment:
We already have the corresponding bbappend, with AUTOREV and other stuff. Maybe it's better to put these variables there? We do not have to add something new, it's already present in sources.

Reply:
sometimes presents , sometimes - not :( here i would like to be sure, that the up to date version will be taken